styles/styles.css

```diff
@@ -818,6 +818,21 @@ main .section.responsive-title h1 {
   font-size: 16px;
 }
 
+.redesign-v2 .icon {
+  display: inline-flex;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.redesign-v2 .icon svg {
+  height: 1em;
+  width: 1em;
+}
+
+.redesign-v2 .section.section-dark {
```

Comment:
Can this be modified to BEM? `section--dark` instead of `section-dark`

Reply:
I changed it to `dark-background`. I missed that dark background class is in the code already.
BTW there is no option to have classes with `--` when they come from the document because everything is converted to the same value:
`dark background`-> `dark-background`
`dark-background` -> `dark-background`
`dark--background` -> `dark-background`
Thats how Franklin is handling it.